src/components/ActivityForm/utils.ts

```diff
@@ -0,0 +1,76 @@
+import * as yup from "yup";
```

Comment:
This is fine, but do have a look at the zod library. You can even convert typescript interfaces to zod
https://transform.tools/typescript-to-zod

Reply:
For the next time that is.

Reply:
Thanks for the comment David, I’ll try the zod library next time